cmd/insert/action/flag.go

```diff
@@ -8,28 +8,24 @@ import (
 	"github.com/spf13/cobra"
 )
 
-const (
-	allActions = "all"
-)
```

Comment:
This was a left over from copying command code. We do not need the support for `-a all` in the new `insert` command.